sql/src/main/java/org/elasticsearch/action/bulk/BulkShardProcessor.java

```diff
@@ -121,11 +123,16 @@ public BulkShardProcessor(ClusterService clusterService,
         this.bulkSize = bulkSize;
         this.createIndicesBulkSize = Math.min(bulkSize, MAX_CREATE_INDICES_BULK_SIZE);
 
-        this.autoCreateIndex = new AutoCreateIndex(settings);
```

Comment:
shouldn't the settings that you get here already include te auto_create_index setting because it is set in the SQLPlugin?